cosmos_rl/dispatcher/status.py

```diff
@@ -755,7 +758,7 @@ def filter_outdated_rollouts(self, rollouts: List[Rollout]) -> List[Rollout]:
             )
             if (
                 estimated_step - rollout.weight_version
-                <= self.config.train.train_policy.allowed_outdated_steps
+                < self.config.train.train_policy.allowed_outdated_steps
```

Comment:
This should be <= rather than <? Since if  allowed_outdated_steps is 0 which means on-policy, here will never append?